src/components/IconButton/IconButton.scss

```diff
@@ -0,0 +1,163 @@
+@use '../../styles/theme' as theme;
+
+.zui-iconButton {
+  border-radius: 50%;
+  padding: 2px;
+  line-height: 0px;
+
+  cursor: pointer;
+  background: none;
+  border: none;
+  display: inline-block;
+  margin: 0px;
+
+  color: theme.$color-greyscale-12;
+
+  &:hover {
+    background-color: theme.$color-greyscale-transparency-3;
+  }
+
+  // Variants
+  &-variant-primary {
```

Comment:
nit: it can be handy to use mix-ins or variables for variants, just so you can define the values for each variant once, then apply the properties once. Not necessary for this PR though!

Reply:
will keep in mind, thanks